include/config/config_world.h

```diff
@@ -70,18 +65,6 @@ STATIC_ASSERT(((EXTENDED_BOUNDS_MODE >= 0) && (EXTENDED_BOUNDS_MODE <= 3)), "You
 
 // The amount of cells in each axis in an area.
 #define NUM_CELLS                   (2 * LEVEL_BOUNDARY_MAX / CELL_SIZE)
-// The maximum amount of collision surfaces (static and dynamic combined)
-#define SURFACE_POOL_SIZE           (LEVEL_BOUNDARY_MAX / 2) // Vanilla: 2300
```

Comment:
I feel like this could have been repurposed (with a rename of course) for the dynamic pool size, so that it would be in a familiar place to people who have messed with the pool size in the past.